lib/matplotlib/tests/test_scale.py

```diff
@@ -74,6 +74,19 @@ def test_extra_kwargs_raise():
         ax.set_yscale('log', nonpos='mask')
 
 
+def test_logscale_invert_transform():
+    fig, ax = plt.subplots()
+    ax.set_yscale('log')
+    # get transformation from data to axes
+    tform = (ax.transAxes + ax.transData.inverted()).inverted()
+
+
+def test_logscale_transform_repr():
+    fig, ax = plt.subplots()
+    ax.set_yscale('log')
+    s = repr(ax.transData)
```

Comment:
show we `assert` anything there?

Reply:
ping @fkloosterman - can this test do more than just make sure it runs?

Reply:
you mean something like:
`assert repr(matplotlib.scale.Log10Transform(nonpos='clip'))=="Log10Transform('clip')"`?

Reply:
@fkloosterman yes something like that. Note, the pep8 checker will complain about the last line unless you put spaces around the `==` operator:
`assert s == "Log10Transform('clip')"`